src/main/java/com/fewlaps/quitnowcache/QNCache.java

```diff
@@ -195,4 +198,29 @@ private String getEffectiveKey(String key) {
         }
         return key;
     }
+    
+    /**
+     * Find 
+     */
+    List<String> findStartsWith(String someCuquiWord) {
```

Comment:
We don't want to provide any way to get dead elements. It could drive to confusion... I suggest removing this method.